content/en/docs/cloud/_index.md

```diff
@@ -28,55 +28,57 @@ overlay [virtual application network](https://netprototalk.com/2019/11/12/virtua
 4. **Developer familiarity**: feels like the frameworks and tools you already know and love (Docker,
    Kubernetes, Terraform, GitOps, etc).
 
-Built by developers for developers, we focus on beautifully crafted interfaces (web and cli),
+   <!-- TODO Consider rewording this to also target decision makers?>
```

Comment:
This might show up in the inspector.

Reply:
Comments? @neilotoole doe comments show up in browser post linting/compiling.

Reply:
resolved by removing all comments of mine.